scripts/generate-hotfix-metadata/run.sh

```
@@ -0,0 +1,14 @@
+#!/bin/bash
```

running with `-e` will ensure that we do not silently fail in this script